rxbinding/src/main/java/com/jakewharton/rxbinding/widget/RxTextView.java

```diff
@@ -219,6 +219,21 @@
     };
   }
 
+  /**
+   * An action which sets the color property of {@code view} color resource IDs.
```

Comment:
This API doesn't accept a resource ID from what I remember, but an actual encoded color as an integer.